### DISCUSSION TITLE
iText scale problem

### DISCUSSION THREAD

**Discussion by jiewang010203** [fabricjs/fabric.js#8960] [Q&A]
I have encountered a problem when using Fabric.js to render text. When I change the values of scaleX and scaleY of an IText object, for example, when increasing them simultaneously (which are always kept the same), I find that the text will appear to jitter and shift up and down from its original position. Is it possible that this problem is caused by precision issues during the calculations?

**Comment by asturur** [comment]
it shouldn't be.
Are you doing this scaling with some custom code.
Can you show it?
